DepositeCalcTests/Tests/LoginPageTests.cs

```diff
@@ -46,12 +32,12 @@ public void NegativeTest(string login, string password)
         public void ValidLoginTest()
         {
             // Arrange
-            var loginPage = new LoginPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
             String expectedUrl = "https://localhost:5001/Calculator";
 
             // Act
             loginPage.Login();
-            Thread.Sleep(500);
+            new WebDriverWait(driver, TimeSpan.FromSeconds(5)).Until(_ => calculatorPage.IsOpened());
```

Comment:
1. Please move this wait to `WaitForReady` method of CalculatorPage class.
2. Please return CalculatorPage in `Login` method. What do you think?
3. Assert calculatorPage.IsOpened(), ok?